root/etc/s6-overlay/s6-rc.d/init-bookstack-config/run

```diff
@@ -4,7 +4,7 @@
 if [ -d /app/www-tmp ]; then
     echo "New container detected. Setting up app folder and fixing permissions."
     mv /app/www-tmp /app/www
-    chown -R abc:abc /app/www
+    lsiown -R abc:abc /app/www
 fi
```

Comment:
This whole bit is unnecessary. The only bit of /app/www that needs to be writable by the user is /app/www/storage - we don't need to do the whole rename/move/chown thing. Need to remove the corresponding section from the Dockerfiles
```
  echo "**** overlay-fs bug workaround ****" && \
  mv /app/www /app/www-tmp && \
  ```
  And then add `/app/www/storage` to the final chown.

Reply:
Resolve in https://github.com/linuxserver/docker-bookstack/pull/158/commits/288cb8d25be6c9f6a248a4b1b44254471f8fe3ba

Reply:
Missed a bit, build should be fixed in https://github.com/linuxserver/docker-bookstack/pull/158/commits/4b7807780be579f3eaffb0017784ab9c7f662e17